server/models/Board.js

```diff
@@ -1,29 +1,108 @@
 const mongoose = require("mongoose");
+const Column = require("../models/Column");
+const Card = require("../models/Card");
 
-const boardSchema = new mongoose.Schema({
-  name: {
-    type: String,
-    required: true,
+const boardSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    columns: [
+      { type: mongoose.Schema.Types.ObjectId, ref: "column", required: false },
+    ],
```

Comment:
In the board schema methods, the way we work with schemas is by utilizing embedded store and not referenced as we are stating the columns will be reference (array of object ids). This can be changed to reflect how we actually are storing the columns which is embedded `column: [columnSchema]`

Reply:
Not sure that I understand what to change this to in this case @moffatethan . I changed the lines above to the line below and the server is crashing:
`
columns: [columnSchema],`

Reply:
@street-jackal Looks like Mongoose handles this for us by not storing it as an embedded document, nevermind! All works fine 👍🏻